lib/achievements.coffee

```diff
@@ -9,6 +9,8 @@ module.exports =
       else
         new Achiever()
 
+    console.info "hellooooo"
```

Comment:
Done, if find an issue in one of the specs, I can take a look at it in a few hours if you can't.